crates/atuin/README.md

```diff
@@ -1 +0,0 @@
-../../README.md
```

Comment:
It's not related to the translation itself, but the symbolic link `crates/atuin/README.md` is replaced by a regular file, which is a copy of the target file. I believe this is not intentional.